extensions/src/main/java/org/pentaho/platform/web/http/api/resources/ActionResource.java

```diff
@@ -131,11 +130,11 @@ protected CallableAction createCallable( final String actionId, final String act
    *
    * @return the {@link IActionInvoker}
    */
-  IActionInvoker getActionInvoker() {
+  IActionInvoker getActionInvoker( ) {
```

Comment:
Why the change? The original code did not violate checkstyle. Also, in general, it is best to avoid checkstyle changes unrelated to the code you are changing directly, which making functional code changes, because it makes the review longer and increases the possibility of merge conflicts.